apps/passport-client/components/screens/ScannedTicketScreens/PodboxScannedTicketScreen/sections/actions/PodboxTicketActionErrorSection.tsx

```diff
@@ -7,9 +10,12 @@ import {
 } from "../../PodboxScannedTicketScreen";
 
 export function PodboxTicketActionErrorSection({
-  error
+  error,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  uiConfig
 }: {
   error: PodboxTicketActionError;
+  uiConfig?: ActionScreenConfig;
```

Comment:
why do we need this?

Reply:
don't strictly need it but i thought i'd leave some breadcrumbs for later